dll/cpl/powercfg/powershemes.c

```diff
@@ -830,6 +846,7 @@ PowerSchemesDlgProc(
                 // TODO:
                 // Initialization failed
                 // Handle error
+                DPRINT1("Pos_InitData failed\n");
```

Comment:
Is a DPRINT1 needed since there is already a MessageBox ?

Reply:
Good point, useless => removed